Build with go install fails

Building from source with `go install` fails:
trying to followi instructions from: https://github.com/dmacvicar/terraform-provider-libvirt#building-from-source

```
$ git clone https://github.com/dmacvicar/terraform-provider-libvirt.git
$ cd terraform-provider-libvirt
$ go install
main.go:5:2: cannot find package "github.com/hashicorp/terraform/plugin" in any of:
	/usr/local/go/src/github.com/hashicorp/terraform/plugin (from $GOROOT)
	/home/anders/go/src/github.com/hashicorp/terraform/plugin (from $GOPATH)
```
go version: go1.9.1 linux/amd64
os: Ubuntu 16.04

I also tried to build version v0.2, same error
```
$ git checkout v0.2
$ go install
main.go:5:2: cannot find package "github.com/hashicorp/terraform/plugin" in any of:
	/usr/local/go/src/github.com/hashicorp/terraform/plugin (from $GOROOT)
	/home/anders/go/src/github.com/hashicorp/terraform/plugin (from $GOPATH)
```

--- a/README.md
+++ b/README.md
@@ -61,8 +61,13 @@ takes advantage of features available only inside of the latest stable release.
 
 You need also need libvirt-dev(el) package installed.
 
-Run `go install` to build the binary. You will now find the binary at
-`$GOPATH/bin/terraform-provider-libvirt`.
+```console
+go get github.com/dmacvicar/terraform-provider-libvirt
+cd $GOPATH/src/github.com/dmacvicar/terraform-provider-libvirt
+go install
+```
+
+You will now find the binary at `$GOPATH/bin/terraform-provider-libvirt`.
 
 
 ## Using the provider

